lib/game/platform_manager.dart

```diff
@@ -134,99 +134,102 @@ class PlatformManager extends Component with HasGameRef<DoodleDash> {
   // that distance above the current highest platform
   double _generateNextY() {
     // Adding platformHeight prevents platforms from overlapping.
-    final currentHighestPlatformY = platforms.last.center.y + platformHeight;
+    final currentHighestPlatformY =
+        _platforms.last.center.y + _tallestPlatformHeight;
 
-    // TODO (Khanh): Switch to difficulty level logic,
-    // increase level of difficulty every 20 or so platforms
     final distanceToNextY = minVerticalDistanceToNextPlatform.toInt() +
-        random
+        _rand
             .nextInt((maxVerticalDistanceToNextPlatform -
-                    minVerticalDistanceToNextPlatform -
-                    100)
+                    minVerticalDistanceToNextPlatform)
                 .floor())
             .toDouble();
 
     return currentHighestPlatformY - distanceToNextY;
   }
 
-  @override
-  void update(double dt) {
-    // Adding Platform Height will ensure that 2 platforms don't overlap.
-    final topOfLowestPlatform = platforms.first.position.y + platformHeight;
+  // Return a platform.
+  // The percent chance of any given platform is NOT equal
+  Platform _semiRandomPlatform(Vector2 position) {
+    // Get a number from 1 to 100 (incl)
+    var nextInt = _rand.nextInt(100) + 1;
 
-    final screenBottom = gameRef.player.position.y +
-        (gameRef.size.x / 2) +
-        gameRef.screenBufferSpace;
+    if (nextInt.between(1, 25)) {
```

Comment:
this entire if could probably be removed since we return the normal platform as the default at the end?